.github/workflows/cmake.yml

```
@@ -0,0 +1,62 @@
+name: "CMake Release"
+
+on:
+  [push, workflow_dispatch]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        include:
+            - name: Windows Tiles Sounds x64 MSVC
```

This should specify cmake to disambiguate between the already-existing msvc job that uses msbuild, which is going to be the usual association imo.
```suggestion
            - name: Windows Tiles Sounds x64 MSVC Cmake
```

---

Will review all of this in a future PR, thanks.